app/mutations/task/comments/updated.rb

```diff
@@ -12,7 +12,8 @@ def execute
   private
 
   def message_params
-    { changed_fields: changed_fields.map { |key| Task.human_attribute_name(key) }.to_sentence }
+    klass = task.is_a?(Supply) ? Supply : Task
```

Comment:
Not really happy with this, I think we need to factor out "Taskable" code in a few places at some point. Ralf wants to redo the supplies at some point anyhow, I'll mention it to him that that may be a good opportunity for refactoring.